pages/ksphere/kommander/1.2/licensing/index.md

```diff
@@ -9,20 +9,20 @@ menuWeight: 6
 
 ## Purchase a License
 
-You can evaluate Kommander running two clusters for free. Kommander cluster plus one additional cluster. If you want to add more clusters you need a valid license.
+You can evaluate Kommander running two clusters for free. Kommander cluster plus one additional cluster. If you want to add more clusters, you need a valid license.
 
-To obtain a valid Kommander license, contact your sales representative at <sales@d2iq.com>. After purchase, download the license file from the [support website](https://support.d2iq.com/s/downloads) using your [login credentials](https://support.d2iq.com/s/login/) to a place accessible by Kommander.
+To obtain a valid Kommander license, contact your sales representative at <sales@d2iq.com>. After purchase, download the license file from the [support website][support-downloads] using your [login credentials][support-creds] to a place accessible by Kommander.
 
 ## Enter a Valid License Key
 
 After you have downloaded the license, an administrator must add it to Kommander.
 
 In the Kommander UI, do the following:
 
-1. Select **Global** in the header drop-down.
-2. Select **Administration** > **Licensing**.
+1. Select **Global** in the workspace header drop-down.
+2. Along the left rail, select **Administration** > **Licensing**.
```

Comment:
I think we usually refer to this as side menu but this works too :)

Reply:
Thanks @natmegs